.docs/user-guide/installation.md

```diff
@@ -40,6 +40,20 @@ specify `stable`, see the additional version names below that are also valid.
 curl -sSL https://dl.bintray.com/emccode/rexray/install | sh -s -- stable
 ```
 
+#### Install a specific version
+
+You can forcibly install a particular version like this:
```

Comment:
I do not agree with the principle of avoiding the word "you". 

For example,  The use of "You" not only complies, but is recommend in the Kubernetes style [guide](https://kubernetes.io/docs/home/contribute/style-guide/#address-the-reader-as-you) for Kubernetes, 

The Mesos project style guide is simply a statement that the project uses the Google style guide. This seems to be a recognition of the Google guide as a quality authority on documentation.

From a perspective going beyond the container space, as part of the Material Design effort, Google maintains a huge, respected and well funded staff to research and test design principles. The [guide](https://material.io/guidelines/style/writing.html#writing-language) for authors of apps running on google platforms may prove to be useful. It specifically recommends the word "you".

The Docker Inc documentation style guide incorporates the AP style guide by reference. The AP guide is paywalled, but in reference to the AP guide, Docker's [style guide](https://docs.docker.com/opensource/doc-style/) says this: "If **you** don't have a copy of (or online subscription to) the AP guide, **you** can almost always find an answer to a specific question by searching the web." My conclusion, Docker probably likes "you", although I confess I didn't spend money to confirm this 100% 

The CloudFoundry style [guide](http://docs.pivotal.io/cf_style.html#audience) explicitly says: you should use "you."

I do not agree with this request, and invite you to reconsider your position.
The guide appears to be contradict mainstream documentation style guides within the container scheduler space.
The word forcibly has been removed.